tests/test_box_mesh.py

```diff
@@ -134,41 +158,73 @@ def _check_triangle_on_boundary(
     return False
 
 
-def test_given_box_mesh__construct_must_find_center_corners_edges_faces_node_sets(
-    default_box_mesh,
+@pytest.mark.parametrize(
+    "box_mesh",
+    [
+        "default_box_mesh",
+        "expanded_box_mesh_x",
+        "expanded_box_mesh_y",
+        "expanded_box_mesh_z",
+    ],
+)
+def test_given_box_mesh_construct_must_find_center_corners_edges_faces_node_sets(
+    box_mesh: BoxMesh, request

```

Comment:
`request` parameter should be typed as `FixtureRequest`

`from pytest import FixtureRequest`